blobfile/_gcp.py

```diff
@@ -229,7 +229,7 @@ def generate_signed_url(
     ordered_headers = sorted(h.items())
     for k, v in ordered_headers:
         lower_k = str(k).lower()
-        strip_v = str(v).lower()
+        strip_v = str(v).strip()

```

Comment:
While this seems like an obvious fix, it's not clear to me which is correct since it was copied and pasted from the google example: https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/storage/signed_urls/generate_signed_urls.py#L67